<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QWidget" name="VideoExcess_Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoExcess - Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>10</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item row="1" column="0">
    <widget class="QPushButton" name="goBackButton">
     <property name="styleSheet">
      <string>
      QPushButton {
        background-color: #6200EA; /* Vibrant Purple */
        color: white;
        font-size: 18px;
        border-radius: 12px; /* Increased roundness */
        padding: 12px;
        border: none; /* Remove the border */
      }
      QPushButton:hover {
        background-color: #3700B3; /* Darker purple when hovered */
        transform: translateY(-2px); /* Slight lift effect */
      }
      QPushButton:pressed {
        background-color: #230036; /* Darker on press */
      }
      </string>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="margin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="modeLabel">
           <property name="text">
            <string>MP3 only mode:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="mp3CheckBox">
           <property name="text">
            <string>MP3</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="playbackSpeedLabel">
         <property name="text">
          <string>Playback Speed:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="playbackSpeedValue">
           <property name="text">
            <string>(0.5x)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="playbackSpeedValue_2">
           <property name="text">
            <string>(1.0x)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="playbackSpeedValue_3">
           <property name="text">
            <string>(2.0x)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSlider" name="playbackSpeedSlider">
         <property name="maximum">
          <number>2</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="styleSheet">
          <string>
          QSlider::groove:horizontal {
            background: #E0E0E0; /* Light background */
            height: 8px; /* More height */
            border-radius: 4px; /* Rounded groove */
          }
          QSlider::handle:horizontal {
            background: #6200EA; /* Handle color */
            width: 16px; /* Wider handle */
            margin: -4px 0; /* Center it */
            border-radius: 8px; /* Rounded handle */
          }
          </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="margin">
        <number>10</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="percentCompleteLabel">
           <property name="text">
            <string>Completion Percentage:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="playbackSpeedValue_4">
             <property name="text">
              <string>(1%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="playbackSpeedValue_7">
             <property name="text">
              <string>(25%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="playbackSpeedValue_5">
             <property name="text">
              <string>(50%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="playbackSpeedValue_6">
             <property name="text">
              <string>(80%)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="playbackSpeedValue_8">
             <property name="text">
              <string>(100%)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QSlider" name="completionSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>80</number>
             </property>
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="styleSheet">
              <string>
              QSlider::groove:horizontal {
                background: #E0E0E0; /* Light background */
                height: 8px; /* More height */
                border-radius: 4px; /* Rounded groove */
              }
              QSlider::handle:horizontal {
                background: #6200EA; /* Handle color */
                width: 16px; /* Wider handle */
                margin: -4px 0; /* Center it */
                border-radius: 8px; /* Rounded handle */
              }
              </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
